source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.WinUI.Mobile/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="SkiaSharp" android:versionCode="1" android:versionName="1.0">
```

Comment:
This file should not be in a library.

Reply:
Nor the info.plists.

Reply:
msbuild was failing without these two files unfortunately, not sure if it is still doing this

Reply:
@anpin I can take over this PR and finish it, if that's fine with you, though I'll have to re-create it and I'll keep the history.

Reply:
@jeromelaban I don't mind that as my attention shifted on something else and I no longer need this actually

Reply:
@anpin Thanks! I'll open another PR then.